src/backend/queries.rs

```diff
@@ -49,7 +49,7 @@ fn features() {
     read(|state| {
         let ids: Vec<PostId> = parse(&arg_data_raw());
         reply(
-            features::features(state, ids)
+            features::features(state, &ids)
```

Comment:
Could you please post the reproduction steps?

Reply:
I deployed this version, and the issue started occurring.

Reply:
Maybe you are upgrading from a wrong version? Since we skipped the previous release,you need to upgrade from the currently deployed version - 29-08-24.